kornia/contrib/distance_transform.py

```diff
@@ -0,0 +1,110 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+import kornia
+
+
+def make_cdt_kernel(
+    kernel_size: int,
+) -> torch.Tensor:
+    # Value of h is derived from the parameters and reference code given by the authors who proposed the algorithm.
+    h = -0.35
+
+    grid = kornia.utils.create_meshgrid(kernel_size, kernel_size, normalized_coordinates=False)

```

Comment:
for internal development and avoid circular package dependencies we tend to do
```suggestion
    grid = create_meshgrid(kernel_size, kernel_size, normalized_coordinates=False)

```
then `from kornia.utils import create_meshgrid`